index.bs

```diff
@@ -1100,6 +1100,15 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
             1. Return |constructCredentialAlg| and terminate this algorithm.
 
+        :   If the [=client=] concludes that it is impossible for any eligible [=authenticator=] to appear,
+        ::  The client MAY prompt the user for [=user consent|consent=] to [=authentication|authenticate=] to the [=[RP]=]. If
+            the user does [=user consent|consent=], the [=client=] MAY return a {{DOMException}} whose name is
```

Comment:
should "user does" actually be "user does not" ?

Reply:
No, this is intended to mean "if the user indeed consents"; so perhaps that needs to be reworded. My thinking is that if the user does not consent, then the client must wait for the timeout instead of returning early.